### PR TITLE
Add Catppuccin Theme

### PR DESCRIPTION
**Pull request by IsotoxalDev**
Add the soothing pastel color theme to helix
repository: https://github.com/catppuccin/helix

image

### CODE REVIEW

File: runtime/themes/catpuccin.toml
Comment (the-mikedavis) [inline runtime/themes/catpuccin.toml:9]:
```suggestion
"variable.other.member" = "green"
```

This was recently fixed in the default theme too: https://github.com/helix-editor/helix/commit/682bcc3a32d90bfb58872610af20f47ee4089c83

Comment (IsotoxalDev) [reply runtime/themes/catpuccin.toml:9]: oh!

**Review by the-mikedavis — APPROVED** [review]
Looks good! 🎨

Could you add the screenshot to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Comment by IsotoxalDev** [comment]
What should i link the theme name to when it is not merged?

**Comment by the-mikedavis** [comment]
Once it is merged in, it will be

```
https://github.com/helix-editor/helix/blob/master/runtime/themes/catpuccin.toml
```

Actually we might want to hold off on updating the wiki until this is merged. I don't have the permissions to merge because of a recent CI change (the required CI checks need an update) so it might take a little while.

**Comment by IsotoxalDev** [comment]
ok

**Pull request merged**
